schematic/schemas/commands.py

```diff
@@ -56,7 +57,16 @@ def schema():  # use as `schematic model ...`
     metavar="<OUTPUT_PATH>",
     help=query_dict(schema_commands, ("schema", "convert", "output_jsonld")),
 )
-def convert(schema, data_model_labels, output_jsonld):
+@click.option(
```

Comment:
For backwards and forwards compatibility can we let --output_jsonld be named --output_path as well? can you look if thats possible? We could then depreciate --output_jsonld over time.